services/CollectionService/src/index.php

```diff
@@ -67,40 +49,127 @@
 });
 
 /**
- * Collection POST route test. Does nothing more than redirect from collection to mounted 
- * takes 'rx' and/or 'checksum' as optional query arguments
+ * Collection POST route. takes $id (valid UUID or empty) for the parent resource as first value to match
+ * takes 'rx' as optional query arguments
  */
```

Comment:
I'm not good at comments nor english. I followed the same type of comments i did for resources. Any suggestion on how to document this is appreciated.